### ISSUE TITLE
[ERROR:flutter/shell/platform/android/platform_view_android_jni_impl.cc(43)] java.lang.IllegalArgumentException: array == null 

### ISSUE THREAD

**Issue by JuanTop** [flutter/flutter#70803]
[ERROR:flutter/shell/platform/android/platform_view_android_jni_impl.cc(43)] java.lang.IllegalArgumentException: array == null
11-18 23:08:50.530  9940  9940 E flutter : 	at android.opengl.Matrix.multiplyMM(Native Method)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.view.AccessibilityBridge$SemanticsNode.updateRecursively(SourceFile:5)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.view.AccessibilityBridge$SemanticsNode.updateRecursively(SourceFile:32)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.view.AccessibilityBridge$SemanticsNode.access$4300(SourceFile:1)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.view.AccessibilityBridge.updateSemantics(SourceFile:23)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.view.AccessibilityBridge$1.updateSemantics(SourceFile:2)
11-18 23:08:50.530  9940  9940 E flutter : 	at io.flutter.embedding.engine.FlutterJNI.updateSemantics(SourceFile:3)
11-18 23:08:50.530  9940  9940 E flutter : 	at android.os.MessageQueue.nativePollOnce(Native Method)
11-18 23:08:50.530  9940  9940 E flutter : 	at android.os.MessageQueue.next(MessageQueue.java:323)
11-18 23:08:50.530  9940  9940 E flutter : 	at android.os.Looper.loop(Looper.java:135)
11-18 23:08:50.530  9940  9940 E flutter : 	at android.app.ActivityThread.main(ActivityThread.java:5658)
11-18 23:08:50.530  9940  9940 E flutter : 	at java.lang.reflect.Method.invoke(Native Method)
11-18 23:08:50.530  9940  9940 E flutter : 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:851)
11-18 23:08:50.530  9940  9940 E flutter : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:741)
11-18 23:08:50.530  9940  9940 E flutter :

**Comment by JuanTop** [comment]
Some mobile applications start to crash

**Comment by pedromassangocode** [comment]
Hi @JuanTop 
Can you please provide your `flutter doctor -v`, your `flutter run --verbose` and a minimal complete reproducible code sample?
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by christopherfujino** [comment]
I can reproduce this while running the mac/android microbenchmarks test locally.

**Comment by christopherfujino** [comment]
Nevermind, re-closing as a duplicate of https://github.com/flutter/flutter/issues/31139

**Comment by RFlame** [comment]
I can reproduce.
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.15.7 19H524, locale
    zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
[!] Android Studio (version 4.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[!] Connected device

! Doctor found issues in 2 categories.

**Comment by RFlame** [comment]
I can reproduce.
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.15.7 19H524, locale
    zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
[!] Android Studio (version 4.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[!] Connected device

! Doctor found issues in 2 categories.

**Comment by christopherfujino** [comment]
> I can reproduce.
> Doctor summary (to see all details, run flutter doctor -v):
> [✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.15.7 19H524, locale
> zh-Hans-CN)
> 
> [✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
> [✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
> [!] Android Studio (version 4.1)
> ✗ Flutter plugin not installed; this adds Flutter specific functionality.
> ✗ Dart plugin not installed; this adds Dart specific functionality.
> [!] Connected device
> 
> ! Doctor found issues in 2 categories.

This issue was closed as a duplicate of https://github.com/flutter/flutter/issues/31139

**Comment by chinmaygarde** [comment]
@christopherfujino I am a bit puzzled about how to prioritize this after you reopened it. There is no reproducible test case linked here. The [issue you linked has been closed](https://github.com/flutter/flutter/issues/31139) as well. Can you add more context here please?

**Comment by christopherfujino** [comment]
> @christopherfujino I am a bit puzzled about how to prioritize this after you reopened it. There is no reproducible test case linked here. The [issue you linked has been closed](https://github.com/flutter/flutter/issues/31139) as well. Can you add more context here please?

I am puzzled too. Closing for a lack of a context (from past me).
